eng/pipelines/common/xplat-setup.yml

```diff
@@ -99,12 +99,12 @@ jobs:
     ${{ if eq(parameters.jobParameters.pool, '') }}:
       pool:
         # Public Linux Build Pool
-        ${{ if and(in(parameters.osGroup, 'Linux', 'FreeBSD'), eq(variables['System.TeamProject'], 'public')) }}:
+        ${{ if and(in(parameters.osGroup, 'Android', 'Linux', 'FreeBSD'), eq(variables['System.TeamProject'], 'public')) }}:
```

Comment:
Maybe leave this as it was to keep using the hosted pools for Android builds? I noticed this will really slow down the machine provisioning.